src/inventory_ui.cpp

```diff
@@ -79,7 +79,12 @@ void debug_print_timer( startup_timer const &tp, std::string const &msg = "inv_u
     add_msg_debug( debugmode::DF_GAME, "%s: %i ms", msg, ( tp_now - tp ).count() );
 }
 
-using item_name_t = std::pair<std::string, std::string>;
+// using item_name_t = std::pair<std::string, std::string>;
+struct item_name_t {
+    std::string sortkey;
```

Comment:
Existing art uses `sort_key`, idk if you want to standardize on `sortkey` or `sort_key` but would be nice to pick one.